analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -90,6 +90,96 @@ open class Npm(
         ) = Npm(managerName, analysisRoot, analyzerConfig, repoConfig)
     }
 
+    companion object {
+        /**
+         * Parse information about licenses from the [package.json][json] file of a module.
+         */
+        internal fun parseLicenses(json: JsonNode): SortedSet<String> {
```

Comment:
IIRC the outcome of a discussion in one of the last developer meetings was that we should only put public static functions into companion objects for discoverability, and move static function with narrower visibility to the top-level. Do I recall that correctly @fviernau?

Reply:
I remember the discussion, but am not exactly sure about the final conclusion. What I want to achieve here is a safe reuse of these functions between the `Npm` and the `NpmDependencyHandler` classes, but without polluting the package namespace. The functions have quite generic names like `parseLicenses()`; otherwise, they would have to be renamed to something more specific, e.g. `parseNpmLicenses()` - I prefer the approach with the companion object in this case.

Reply:
I agree. So from my point of view this can stay as-is, even if it might differ a bit from the rule of thumb we agreed on.